frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -52,23 +56,16 @@ body {
 }
 
 .ThemeNight.EmbedFrame .fullscreenNightText {
-  color: var(--mb-color-text-white-alpha-85);
   transition: color 1s linear;
 }
 
 .ThemeNight.EmbedFrame svg text {
-  fill: var(--mb-color-text-white-alpha-85) !important;
   stroke: none !important;
 }
 
 .ThemeNight.EmbedFrame .DashCard .Card {
+  border: 1px solid var(--mb-color-border);
   background-color: var(--mb-color-bg-black);
-  border: 1px solid var(--mb-color-bg-dark);
-}
-
-.ThemeNight.EmbedFrame .enableDotsOnHover :global(.dc-tooltip circle.dot:hover),
-.ThemeNight.EmbedFrame .enableDots :global(.dc-tooltip circle.dot) {
```

Comment:
[Can be removed](https://metaboat.slack.com/archives/C064QMXEV9N/p1720084476549369) because we don't have `dc` in the project anymore.